test/lib/tpunit++.cpp

```diff
@@ -396,6 +403,27 @@ void tpunit::TestFixture::tpunit_detail_do_tests(TestFixture* f) {
     }
 }
 
+void tpunit::TestFixture::testLog(const string& newLog) {
+    // Format the buffer nicely as we print it out.
+    testOutputBuffer = testOutputBuffer + "[              ]    " + newLog + "\n";
```

Comment:
The formatting here won't look right now that we've removed `[              ]` from the rest of the tests.